init.lua

```diff
@@ -87,7 +87,7 @@ end
 
 function ts_doors.register_door(item, description, texture, sounds, recipe)
 	if not minetest.registered_nodes[item] then
-		minetest.log("[ts_doors] bug found: "..item.." is not a registered node. Cannot create doors")
+		minetest.log("error", "[ts_doors] Bug found: "..item.." is not a registered node. Can not create doors.")
```

Comment:
I would remove the `.` sign from the EOL.

Reply:
Why would you do that?
The log message is in the form of natural language and consists of two sentences.
Leaving the dot out from the last sentence will render the whole line looking unfinished in my opinion.